roles/schema/defaults/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+# connectivity variables
+sr_port: 8081  
```

Comment:
This does not seem to be used elsewhere

Reply:
The variable is now removed from the role.